src/clients/api/queries/getIsolatedPools/formatOutput/index.ts

```diff
@@ -146,7 +148,7 @@ const formatToPools = ({
         token: vToken,
       });
       const supplyBalanceTokens = supplyBalanceVTokens.multipliedBy(exchangeRateVTokens);
-      const supplyBalanceCents = +supplyBalanceTokens.multipliedBy(tokenPriceDollars).dividedBy(36);
+      const supplyBalanceCents = +supplyBalanceTokens.multipliedBy(tokenPriceCents).dividedBy(36);
```

Comment:
Why do we divide this value by 36?

`supplyBalanceTokens` should be expressed in tokens (not in vTokens), so converting it to cents should only be a matter of multiplying that value by the token price in cents.